src/Sulu/Bundle/MediaBundle/Resources/js/containers/MultiMediaSelection/MultiMediaSelection.js

```diff
@@ -134,7 +134,7 @@ export default class MultiMediaSelection extends React.Component<Props> {
                                 key={media.id}
                             >
                                 <div className={multiMediaSelectionStyle.mediaItem}>
-                                    {media.thumbnails[THUMBNAIL_SIZE]
+                                    {typeof media.thumbnails !== 'undefined' && media.thumbnails[THUMBNAIL_SIZE]
```

Comment:
I think instead of the `typeof` check I would just go for `media.thumbnails && media.thumbnails[THUMBNAIL_SIZE]`, in order to be more consistent. Or didn't that work for some reason?